community-operators/postgres-operator/postgres-operator.v3.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,352 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Database
+    certified: "false"
+    repository: https://github.com/CrunchyData/postgres-operator
+    containerImage: crunchydata/postgres-operator:centos7-3.5.0
+    createdAt: 2019-02-01T11:59:59Z
+    description: PostgreSQL is a powerful, open source object-relational database system with over 30 years of active 
```

Comment:
This short description is a bit too long, aim for about 135 characters

Reply:
I shortened the description. I did have some confusion about whether this should be a description for the operator or the thing that it manages.